setup_test.go

```diff
@@ -192,6 +192,29 @@ func TestCaddyParse(t *testing.T) {
 				},
 			},
 		},
+		{
+			`
+			txtdirect {
+				enable host
+				redirect https://example.com
+				prometheus {
+					path /metrics
+					path_metrics_whitelist example.com,example.org
```

Comment:
Any specific reasoning making it a list aka delimited by ","?

Fine with this, but wanted to get a bit of context here.

Reply:
If we used space instead, the Caddy controller would've split it into separate values for `path_metrics_whitelist` and we had to use the controller to walk through the values inside the Prometheus parser and check for the end of the scope in there too. Just seemed like an easy solution to not change the parser code anymore.

Reply:
Ack.